cmd/create.go

```diff
@@ -137,104 +197,333 @@ type createConfig struct {
 	// minimum implementation of the signature itself and example tests.
 	Template string
 
-	// Verbose output
-	Verbose bool
-
-	// Confirm: confirm values arrived upon from environment plus flags plus defaults,
-	// with interactive prompting (only applicable when attached to a TTY).
-	Confirm bool
+	// Name of the Function
+	// Not prominently highlighted in help text or confirmation statements as it
+	// is likely this will be refactored away to be part of the Deploy step, in
+	// a git-like flow where it is not required to create, but is part of a named
+	// remote for a one-to-many replationship.
```

Comment:
:thinking:

Reply:
Removed as premature and anxiety-creating...